celery_once/backends/sentinel.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+
+"""Definition of the sentinel locking backend."""
+
+from __future__ import absolute_import
+
+try:
+    from redis.lock import Lock
+except ImportError:
+    raise ImportError(
+        "You need to install the redis library in order to use Redis"
+        " backend (pip install redis)"
+    )
+
+from celery_once.tasks import AlreadyQueued
+
+
+class Sentinel(object):
+    """Sentinel backend."""
+
+    def __init__(self, settings):
+        """
+            settings are like
```

Comment:
Can we clean up this docstring slightly or just remove it?

Reply:
done